scipy/spatial/_spherical_voronoi.py

```diff
@@ -166,20 +166,23 @@ class SphericalVoronoi:
     def __init__(self, points, radius=1, center=None, threshold=1e-06):
 
         if radius is None:
-            radius = 1.
+            radius = 1
```

Comment:
why is this change needed? The type coercion below seems like it would do just fine without changing this.

Reply:
Fixed.